046-kraft-liveness-readiness.md

```diff
@@ -159,7 +160,7 @@ This will be used for the broker only and combined readiness checks later and wi
 ### Phase 2 - Strimzi adds support for broker only and controller only modes
 
 * Controller only pod liveness and readiness checks are fully implemented to match this proposal.
-* Broker only pods are marked as both alive and ready if listening on port 9091.
+* Broker only pods are marked as both alive and ready when there is a Java process running (no change from phase 1).
```

Comment:
Readiness will be based on broker state retrieved from KafkaAgent. This is just in phase 2 implementation where we will be fully implementing the controller liveness and readiness and in phase 3, we will fully implement readiness of broker only and combined mode. If this doesn't look right, we can implement the readiness of broker only mode in phase 2 but I thought since readiness is same for broker only and combined mode, it makes sense to implement them at the same time.

Reply:
I think even in the Phase 2 the readiness probe should be defined properly